extensions/blocks/calendly/calendly.php

```diff
@@ -79,9 +78,14 @@ function set_availability() {
  * @return string
  */
 function load_assets( $attr, $content ) {
+	if ( ! is_available() ) {
```

Comment:
For Simple Payments, we checked if we were not on an RSS feed: 
https://github.com/Automattic/jetpack/blob/7f3bd724c184747db76937e367d1840c7a91fb6f/modules/simple-payments/simple-payments.php#L159

See #13723

Maybe we should do the same here, or integrate that check to the `upgrade_nudge` method?